src/js/components/RangeSelector/doc.js

```diff
@@ -116,8 +116,8 @@ export const themeDoc = {
     defaultValue: 'light-4',
   },
   'rangeSelector.edge.type': {
-    description: 'The edge style type.',
-    type: "'bar' | 'disc'",
+    description: 'The edge control type.',
+    type: "'bar' | 'disc' | node",
```

Comment:
What about typescript?

Reply:
as far as I know, props of the themeDocs are not part of the index.d.ts, only component props.

Reply:
oops, you are right!